scapy/tools/UTscapy.py

```diff
@@ -79,7 +87,7 @@ def retry_test(func):
             success = True
             break
     if not success:
-        six.reraise(t, v, tb)
+        reraise(t, v, tb)
```

Comment:
```suggestion
        raise v.with_traceback(tb)
```
and remove `reraise`